cmd/context/update-kubeconfig_test.go

```diff
@@ -255,78 +256,21 @@ func Test_ExecuteUpdateKubeconfig_EnabledKubetoken(t *testing.T) {
 			okContext := okteto.Context()
 			kubeconfigPaths := []string{file}
 
-			err = ExecuteUpdateKubeconfig(okContext, kubeconfigPaths, okClientProvider)
+			err = NewKubeconfigCMD(okClientProvider).Execute(okContext, kubeconfigPaths)
 			assert.NoError(t, err, "error writing kubeconfig")
 
 			cfg := kubeconfig.Get(kubeconfigPaths)
-			assert.NotNil(t, cfg, "kubeconfig is nil")
+			require.NotNil(t, cfg, "kubeconfig is nil")
 			assert.Equal(t, tt.context.CurrentContext, cfg.CurrentContext, "current context has changed")
 			assert.Equal(t, tt.context.Contexts[tt.context.CurrentContext].Namespace, cfg.Contexts[tt.context.CurrentContext].Namespace, "namespace has changed")
-			assert.NotNil(t, cfg.AuthInfos)
+			require.NotNil(t, cfg.AuthInfos)
```

Comment:
🤔  why change this to require and others left to assert?

Reply:
because if we are just asserting we would get a panic and I find it easier to have a clear error rather than a panic. require stops the execution but assert continue with the execution and after all the test has run it returns the error. It's just to not having a panic on the test if something fails earlier